firebase-firestore/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Unreleased
+
+# 21.0.0
```

Comment:
major version bump due to kotlin nullability changes. (Affects most/all other firebase components too.)